### PR TITLE
fix building of the app for ios simulator from cli/xcode

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -34,12 +34,22 @@ abstract_target 'Status' do
   target 'StatusImPR' do
   end
 
-  use_flipper!({ 'Flipper' => '0.74.0' })
-  post_install do |installer|
-    flipper_post_install(installer)
+  #commented out temporarily
```
Comment (shivekkhurana) [inline ios/Podfile:37]: Is this supposed to stay commented out?

Comment (rasom) [reply ios/Podfile:37]: yep

Comment (rasom) [reply ios/Podfile:37]: at least till someone will need flipper

**Review by shivekkhurana — APPROVED** [review]

File: ios/Podfile (hunk 1 above)
Comment (shivekkhurana) [reply ios/Podfile:37]: I believe Andrey was/is working on this? We can put it back when he is back. Does this break the flow?

**Review by anastasiyaig — APPROVED** [review]
Screenshot 2021-05-11 at 12 39 01

**Pull request merged**